### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by JoranZwet**
Fixed bugs in eval_shape_fun function
Fixed scaling of 2D stiffness matrix for unitz

### CODE REVIEW

File: pymoto/modules/assembly.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def _prepare(self, domain: DomainDefinition, *args,
 
         # Numerical integration
         siz = domain.element_size
-        w = np.prod(siz[:domain.dim]/2)
+        w = np.prod(siz/2)
```
Comment (aatmdelissen) [inline pymoto/modules/assembly.py:216]: I think the original is the correct one. Can you verify this with the test I've added?

File: pymoto/common/domain.py [hunk 2]
```diff
@@ -184,10 +184,10 @@ def eval_shape_fun(self, pos: np.ndarray):
         """
         v = np.prod(self.element_size[:self.dim])
         assert v > 0.0, 'Element volume needs to be positive'
-        ret = np.ones(self.nnodes)/v
+        shapefn = np.ones(self.elemnodes)/v
```
Comment (aatmdelissen) [inline pymoto/common/domain.py:187]: How did the original one ever work?! 😝

**Review by aatmdelissen — CHANGES_REQUESTED** [review]
I've added a uniaxial tensile test for both 2D and 3D. The 3D case works correctly, but the 2D case has some issues. Could you verify this test?

File: pymoto/common/domain.py (hunk 2 above)
Comment (JoranZwet) [reply pymoto/common/domain.py:187]: It didn't, but it was never used anywhere ¯\_(ツ)_/¯

File: pymoto/modules/assembly.py (hunk 1 above)
Comment (JoranZwet) [reply pymoto/modules/assembly.py]: Verified and restored original

**Review by aatmdelissen — APPROVED** [review]

**Pull request merged**